cockroachcloud/security-overview.md

```diff
@@ -5,93 +5,158 @@ toc: true
+    <td>✓</td>
+    <td>Client identity authentication using a username and password</td>
+  </tr>
+  <tr>
+    <td>✓</td>
+    <td>✓</td>
+    <td><a href="../{{site.versions['stable']}}/security-reference/scram-authentication.html">SASL/SCRAM-SHA-256 secure password-based authentication</a></td>
+  </tr>
+  <tr>
+    <td>&nbsp;</td>
+    <td>✓</td>
+    <td>Web console authentication with third-party <a href="../{{site.versions['stable']}}/sso-db-console.html">Single Sign On (SSO)</a> using <a href="https://openid.net/connect/">OpenID Connect OIDC</a> or <a href="https://en.wikipedia.org/wiki/Security_Assertion_Markup_Language">SAML</a></td>
+  </tr>
+  <tr>
+    <td>✓</td>
+    <td>✓</td>
+    <td>SQL Client authentication with <a href="cloud-sso-sql.html">Cluster SSO</a> using CockroachDB Cloud as identity provider</td>
+  </tr>
+  <tr>
+    <td>&nbsp;</td>
+    <td>✓</td>
+    <td>SQL Client authentication with <a href="../{{site.versions['stable']}}/sso-sql.html">Cluster SSO</a> using customer-managed identity providers</td>
+  </tr>
+  <tr>
+    <td>&nbsp;</td>
+    <td>✓</td>
+    <td>Client identity authentication using <a href="client-certs-dedicated.html">PKI certificates</a></td>
+  </tr>
+  <tr>
+    <td>&nbsp;</td>
+    <td>✓</td>
+    <td><a href="../{{site.versions['stable']}}/manage-certs-revoke-ocsp.html">OCSP</a> certificate revocation protocol</td>
+  </tr>
+  <tr>
+    <td>&nbsp;</td>
+    <td>✓</td>
+    <td><a href="../{{site.current_cloud_version}}/cluster-api.html">Cluster API</a> access using login tokens</td>
+  </tr>
+  <tr>
+  <td rowspan="5" >Data Protection</a></td>
+    <td>✓</td>
+    <td>✓</td>
+    <td>Encryption-in-flight using TLS 1.3</td>
+  </tr>
+  <tr>
+    <td>✓</td>
+    <td>✓</td>
+    <td>Automatic backups for AWS clusters are encrypted-at-rest using <a href="https://docs.aws.amazon.com/AmazonS3/latest/dev/UsingServerSideEncryption.html">AWS S3’s server-side encryption</a></td>
+  </tr>
+  <tr>
+    <td>✓</td>
+    <td>✓</td>
+    <td>Automatic backups for GCP clusters are encrypted-at-rest using <a href="https://cloud.google.com/storage/docs/encryption/default-keys">Google-managed server-side encryption keys</a></td>
+  </tr>
+  <tr>
+    <td>✓</td>
+    <td>✓</td>
+    <td>Industry-standard encryption-at-rest provided at the infrastructure level by your chosen deployment environment, such as Google Cloud Platform (GCP), Amazon Web Services (AWS), or Microsoft Azure.
+  </tr>
+  <tr>
+    <td>&nbsp;</td>
+    <td>✓</td>
+    <td><a href="cmek.html">Customer Managed Encryption Keys (CMEK)</a>.
+  </tr>
+  <tr>
+  <td rowspan="4" ><a href="authorization.html">Access Control (Authorization)</a></td>
+    <td>✓</td>
+    <td>✓</td>
+    <td>SQL users with direct privilege management</td>
+  </tr>
+  <tr>
+    <td>✓</td>
+    <td>✓</td>
+    <td>SQL Role-based access control (RBAC)</td>
+  </tr>
+  <tr>
+    <td>✓</td>
+    <td>✓</td>
+    <td>Cloud Organization users with legacy roles</td>
```

Comment:
We should remove this one to avoid confusion. We would deprecate legacy roles soon anyways.

Reply:
ok